scripts/cddl_gen.py

```diff
@@ -1387,6 +1387,9 @@ def str_to_c_code(self, cbor_str, var_name):
         return f'uint8_t {var_name}[] = {{{", ".join(hex(c) for c in cbor_str)}}};\n'
 
 
+xcoder_tuple = namedtuple("xcoder", ["body", "func_name", "type_name"])
```

Comment:
As an aside, I'm a huge fan of `namedtuple` but there is also `typing.NamedTuple` that gives you the same functionality but with type hints and further class customization:

```
class XCoder(typing.NamedTuple):
    body: typing.Any
    func_name: str
    type_name: str
```
You can then use it as a normal class from anywhere.

Reply:
Cool! I'll definitely look into that.

Reply:
I changed it to NamedTuple, thanks. Please take a look.